pkg/apis/workflow/v1alpha1/types.go

```diff
@@ -293,6 +294,9 @@ type Parameter struct {
 	// Name is the parameter name
 	Name string `json:"name"`
 
+	// PassthroughRegex matches multiple parameter names using a regex and passes through their values
+	PassthroughRegex *string `json:"passthroughRegex,omitempty"`
```

Comment:
Am renaming it.

Reply:
Thank you for renaming the variable. Since we renamed this variable, I think renaming related functions and comments in this change helps maintainability.

> Hmm... "passthrough" sounds good in arguments, but a little bit weird in inputs because it doesn't "passthrough" but rather just receiving multiple parameters by regexp. 

What do you think about this?

Reply:
Good point, I've renamed it everywhere else.

I agree is sounds weird in inputs, but I can't think of anything else other than 'wildcard'. I'm open to suggestions ...

Reply:
Do you think defining 2 fields like `NameRegexp` for inputs and `PassthroughNameRegexp` for arguments is easier for users although the implementation will be more complicated?

Reply:
And maybe defining `ArgumentPatameter` and `InputParameter` separately which contains inline `Parameter`?

Reply:
Two fields would be easier for users, I'll do that.

I'm not convinced that it's worth the extra implementation complexity of ArgumentParameter and InputParameter, but happy to go ahead if you think it is.

Reply:
I think having two fields is enough for now. :)